fuzz/ocsp_parse_url.cc

```diff
@@ -0,0 +1,32 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+#include <openssl/mem.h>
+#include <openssl/ocsp.h>
+#include <string>
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *buf, size_t len) {
+  std::string input(reinterpret_cast<const char *>(buf),
+                    std::next(reinterpret_cast<const char *>(buf), len));
```

Comment:
What is the std::next part doing? Does this add a null terminator to the end of the string? Is it safe to pass this to OCSP_parse_url?